src/contexts/betslip.tsx

```diff
@@ -117,7 +118,10 @@ export const BetslipProvider: React.FC<Props> = (props) => {
     })
   }, [ items ])
 
+  const minBet = isLiveBet && appChainType === 'mainnet' ? minLiveBetAmount : undefined
+
   const isAmountLowerThanMaxBet = Boolean(betAmount) && typeof maxBet !== 'undefined' ? +betAmount <= maxBet : true
+  const isAmountBiggerThanMinBet = Boolean(betAmount) && typeof minBet !== 'undefined' ? +betAmount >= minBet : true
 
   const isBetAllowed = isConditionsInCreatedStatus && isComboAllowed && isPrematchBetAllowed && isAmountLowerThanMaxBet
```

Comment:
&& isAmountBiggerThanMinBet

Reply:
missed it, thanks
bf985a0b29f2eb97c4671afca7bc22dd5fc6fc30